### DISCUSSION TITLE
chromeOS 下的 Linux 添加存储失败

### DISCUSSION THREAD

**Discussion by hans-johnson** [AlistGo/alist#6134] [Q&A]
使用 chromeOS 的 Linux 子系统，通过一键脚本成功安装 AList。
再使用非子系统的 Chrome 通过 127.0.0.1 成功登录，之后和我在其他设备上的操作一样，添加阿里云盘 Open，然而显示失败：
> Failed init storage: Post "https://openapi.alipan.com/adrive/v1.0/user/getDriveInfo": dial tcp: lookup openapi.alipan.com on ************93:53: cannot unmarshal DNS message

添加夸克网盘同样失败，显示：
> Failed init storage: Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp: lookup drive.quark.cn on ************93:53: read udp ************02:49383->************93:53: i/o timeout

使用 Linux 子系统下的 Firefox 操作，问题一样。
请问这是否与系统有关？我希望通过 Linux 子系统搭建 AList，通过 Android 子系统上的播放器观影，这是否可行？

**Comment by pongfcnkl** [comment]
可能得用安卓的包

**Reply by hans-johnson** [reply]
不太敢用 Android 下的 WebDAV，一个原因是，我最早用 [阿里云WebDAV](https://github.com/eritpchy/aliyundrive-webdav)，没啥问题，后来转用 AList，也正常。
但是同期开始我的梯子就连不上，怎么都解决不了，最后把整机重置，梯子正常了，结果 AList 不正常了。拿另一个 Android 手机跑 AList，别的设备能连它，就这个 Chromebook 连不上，虽然我也找不出这两者直接关联。

**Reply by pongfcnkl** [reply]
@hans-johnson 
我的意思是在termux里跑安卓的二进制

**Reply by hans-johnson** [reply]
因为上面说了的原因，我不太想在 Android 里弄 AList，我不想再 Powerwash 一遍。

但 1 楼的问题已经解决，给有同样问题的朋友：
在主系统浏览器地址栏输入 chrome://flags/#disable-dns-proxy，设置为 Enabled 后重启，本机可以添加网盘了。

虽然主系统正常，但目前 Linux 或 Android 子系统都无法访问 AList。

**Comment by hans-johnson** [comment]
刷到了这篇：https://bbs.oneplus.com/thread/1371387436365512709
看上去在 Linux 部署 alist，在其他系统下使用是可行的，但我没有成功。
